dockers/docker-dhcp-relay/docker-dhcp-relay.supervisord.conf.j2

```diff
@@ -43,8 +43,16 @@ isc-dhcp-relay-{{ vlan_name }}
 
 
 {# Create a program entry for each DHCP relay agent instance #}
+{% set relay_for_ipv4 = { 'flag': False } %}
 {% for vlan_name in VLAN %}
 {% if VLAN[vlan_name]['dhcp_servers'] %}
+{% for dhcp_server in VLAN[vlan_name]['dhcp_servers'] %}
+{% if  dhcp_server | ipv4 %}
+{% set _dummy = relay_for_ipv4.update({'flag': True}) %}
+{% endif %}
+{% endfor %}
+{% if relay_for_ipv4.flag %}
+{% set _dummy = relay_for_ipv4.update({'flag': False}) %}
```

Comment:
i do not see where this variable ```_dummy``` is used, if it is not used, why do you define it?

Reply:
It's necessary to conform to the syntax of jinja2. Removing the variable '_dummy', the jinja2 throw an syntax error as follows.

File "/home/admin/docker-dhcp-relay.supervisord.conf.j2", line 42, in template
    {% set relay_for_ipv4.update({'flag': True}) %}
jinja2.exceptions.TemplateSyntaxError: expected token 'end of statement block', got '('

Reply:
I believe this is because you are unnecessarily using `set` on these lines. Have you tried simply using the following syntax?

`{% relay_for_ipv4.update({'flag': False}) %}`

Reply:
> I believe this is because you are unnecessarily using `set` on these lines. Have you tried simply using the following syntax?
> 
> `{% relay_for_ipv4.update({'flag': False}) %}`

Yes, I tried before. It also throws an syntax error, as follows.
 {% relay_for_ipv4.update({'flag': False}) %}
jinja2.exceptions.TemplateSyntaxError: Encountered unknown tag 'relay_for_ipv4'. Jinja was looking for the following tags: 'elif' or 'else' or 'endif'. The innermost block that needs to be closed is 'if'.